_drafts/zero-to-main-libc.md

```diff
@@ -0,0 +1,385 @@
+---
+title: "From zero to main(): Bootstrapping libc"
```

Comment:
Would probably work "newlib" somewhere into the title and/or description.

Reply:
:+1: